tests/set-labels/empty-resource/.expected/results.yaml

```diff
@@ -0,0 +1,8 @@
+apiVersion: kpt.dev/v1
+kind: FunctionResultList
+metadata:
+  name: fnresults
+exitCode: 0
+items:
+  - image: gcr.io/kpt-fn/set-labels:unstable
+    exitCode: 0
```

Comment:
For empty resource, I would expect the function can still give a helpful user message, something like no resources found,  or updated 0 labels. So as users can take actions. 
Note: it's hard for users to debug an unexpected behavior of a KRM function, so it's more critical to give user messages.